core/src/main/java/nl/nn/adapterframework/extensions/javascript/J2V8.java

```diff
@@ -35,41 +37,46 @@
 
 	private Logger log = LogUtil.getLogger(this);
 	private V8 v8;
+	private String alias = null;
 
 	@Override
-	public void startRuntime() {
-		startRuntime(null, null);
+	public void setScriptAlias(String alias) {
+		this.alias = alias;
 	}
 
-	/**
-	 * If path is null or empty, it will use the log.dir
-	 * If the log.dir is relative it will turn it into an absolute path
-	 */
-	public void startRuntime(String alias, String path) {
-		String directory = path;
-		if(StringUtils.isEmpty(directory)) {
-			directory = AppConstants.getInstance().getResolvedProperty("ibis.tmpdir");
-		}
-		if(directory != null) {
+	private String getTempDirectory() {
+		String directory = AppConstants.getInstance().getResolvedProperty("ibis.tmpdir");
+
+		if (StringUtils.isNotEmpty(directory)) {
 			File file = new File(directory);
 			if (!file.isAbsolute()) {
 				String absPath = new File("").getAbsolutePath();
-				if(absPath != null) {
+				if (absPath != null) {
 					file = new File(absPath, directory);
 				}
 			}
-			if(!file.exists()) {
+			if (!file.exists()) {
 				file.mkdirs();
 			}
 			String fileDir = file.getPath();
-			if(StringUtils.isEmpty(fileDir) || !file.isDirectory()) {
-				throw new IllegalStateException("unknown or invalid path ["+((StringUtils.isEmpty(fileDir))?"NULL":fileDir)+"], unable to load J2V8 binaries");
+			if (StringUtils.isEmpty(fileDir) || !file.isDirectory()) {
+				throw new IllegalStateException("unknown or invalid path [" + ((StringUtils.isEmpty(fileDir)) ? "NULL" : fileDir) + "], unable to load J2V8 binaries");
 			}
```

Comment:
I dislike these spaces being added purely because your IDE likes it. There is a codestyle which we all have to follow.